### PR TITLE
Tribals & Raiders

### PR DESCRIPTION
**Pull request by ProbablyCarl**
This PR introduces Tribals as their own faction, rather than the moping mess they were prior.
Additionally, it returns Outlaws to being Raiders, adds a leadership role and adjusts their equipment. See below.
- - -
Factions:
-  Raiders:
    - Outlaw renamed to Raider.
    - Outsider introduced as a role, housing the equipment for outcasts, remnants and various other groups that aren't Raider quality.
    - Raider Captain introduced.
    - Raider compound has access requirements.

-  Tribals:
    - Tribal removed.
    - Tribal Hunter added.
    - Tribal Gatherer added. This is the introductory role. You must play it for XP to unlock Hunter, and then play Hunter to get Shaman.
    - Tribal Shaman added. They have all tribal recipes, allowing the tribe to cater to its own. RP wise and otherwise.
    - Tribal camp now has access requirements.
- - -
Balance:
 - Deprecation of Trainshot. It is now unobtanium.
 - Flechettes introduced as a thematic replacement for Trainshot. These do two(2) stamina and brute on hit, with a total of twenty-four(24!!!) projectiles per shot. They have incredible AP, but horrible spread. Lawndart moment.
 - Magnum Buckshot reintroduced. This fills a niche between the AP and downing purpose of flechette, and the outright killing power of buckshot.
 - Neostead alternative introduced. You can no longer get the tube-swapping mechanical Neostead. Instead, you get a general twelve(12) shell capacity shotgun.
 - Tribals can once again get deathclaw spears, in a significantly nerfed state. In comparison to the previous 15% AP, it now has 9% AP. It retains the damage it had prior.
 - Khan Senior Enforcer provided a riot shotgun, rather than the Neostead.
 - Mosin removed from general loot piles.
 - Mosin and Enfield significantly altered to be competitive, albeit niche, bolt actions.
 - Legion have lost their roundstart HMG.
 - Shotgun modifiers returned to the Caravan(3D/5%AP) and Widowmaker(2D/15%AP). They both now have their own unique positions as sidearms / primaries.
- - -
Other:
 - Unique projectiles for F13(primarily NPC projectiles), moved to their own file.
 - Intercoms now have the appropriate sprite. For what little there are of them around the map.
- - -
Map:
 - Various dungeon reworks included. This is to address the low quality loot concerns and the issue of door stacking to deal with a specific boss.
 - Deprecation of 'Radiation' areas, replaced with appropriate areas.. WHY DID THESE EXIST IN THE FIRST PLACE? AAAAAAAAAAAAAAAAAAAAAAAA
 - Watchtower added to the Tribal camp.
 - Warren facelift. Landmines with low alpha have been sprinkled about the map, in a thematic overhaul for the Raiders now being present.
 - Shopkeep stock updated.
 - Probably a lot of other stuff that I forgot.
- - -

### CODE REVIEW

**Comment by ProbablyCarl** [comment]
image
image
image
image
image

File: code/modules/projectiles/guns/ballistic/launchers.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 /obj/item/gun/ballistic/revolver/grenadelauncher
 	desc = "A break-operated grenade rifle. Projectiles travel slowly."
-	name = "grenade rifle"
+	name = "M79 grenade launcher"
```
Comment (CDRDecky) [inline code/modules/projectiles/guns/ballistic/launchers.dm:6]: Thought we were doing away with the realistic gun name convention.

File: code/modules/projectiles/guns/ballistic/launchers.dm [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 /obj/item/gun/ballistic/revolver/grenadelauncher
 	desc = "A break-operated grenade rifle. Projectiles travel slowly."
```
Comment (CDRDecky) [inline code/modules/projectiles/guns/ballistic/launchers.dm:5]:
```suggestion
	desc = "A break-operated M79 grenade launcher. Projectiles travel slowly."
```

**Review by CDRDecky — APPROVED** [review]

File: code/modules/projectiles/guns/ballistic/launchers.dm (hunk 1 above)
Comment (ProbablyCarl) [reply code/modules/projectiles/guns/ballistic/launchers.dm:6]: :(

**Pull request merged**